bindings/python/iota_sdk/client/client.py

```diff
@@ -517,3 +517,8 @@ def submit_payload(self, payload) -> List[HexStr|Block]:
         })
         result[1] = Block.from_dict(result[1])
         return result
+
+    def listen(self, topics: List[str], handler):
```

Comment:
Should we also call this `listen_mqtt` to avoid confusion with the other `listen`, event if it's in the wallet?

Reply:
Done